connect/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# @perspect3vism/ad4m-connect
+
+## 0.3.0
```

Comment:
Should be a patch version upgrade: 0.2.1

Reply:
ah yeah... I guess changset is putting 0.X.0 as minor version bump and X.0.0 as a major version bump.... for now ill just update manually